src/util/http_utils.js

```diff
@@ -671,6 +672,17 @@ function authorize_session_token(req, options) {
         throw new options.ErrorClass(options.error_invalid_token);
     }
 }
+function validate_nsfs_whitelist(req) {
+    const client_ip = parse_client_ip(req);
+    if (config.NSFS_WHITELIST.length === 0) return;
+    for (const whitelist_ip of config.NSFS_WHITELIST) {
+        if (client_ip.includes(whitelist_ip)) {
```

Comment:
shouldn't the condition be client_ip === whitelist_ip?
I mean, what if the client_ip is 11.22.33.44 and the white_list_ip is 11.22.33.4?

Reply:
@romayalon, method `parse_client_ip()` returns the IPs like this `::ffff:127.0.0.1` , thats why I added includes()  instead of `===` , Do I need to get IP only by filtering out the unwanted char?